crates/bevy_diagnostic/src/log_diagnostics_plugin.rs

```diff
@@ -52,6 +58,7 @@ impl Plugin for LogDiagnosticsPlugin {
 }
 
 impl LogDiagnosticsPlugin {
+    /// Filter logging to only the paths in `filter`.
```

Comment:
```suggestion
    /// Filter logging to only the paths in [`Self::filter`].
```

Reply:
I think this rephrasing would be less clear as this function is setting/overriding the current value of `Self::filter`.